app/celery/lookup_recipient_communication_permissions_task.py

```diff
@@ -106,18 +110,5 @@ def recipient_has_given_permission(
         current_app.logger.info('Communication item for recipient %s not found on notification %s',
                                 id_value, notification_id)
 
-        if default_send_flag:
-            return None
-
-        # return status reason message if message should not be sent
-        return "No recipient opt-in found for explicit preference"
-    else:
-        current_app.logger.info('Value of permission for item %s for recipient %s for notification %s: %s',
-                                communication_item.va_profile_item_id if communication_item else None,
-                                id_value, notification_id, is_allowed)
```

Comment:
You moved this back inside the "try".  My apologies if this wasn't clear, but it still belongs here.  I intended to suggest that you delete the "else" and de-indent here.